web-common/src/features/help/initPylonChat.ts

```diff
@@ -1,4 +1,4 @@
-import type { V1User } from "../../client";
+import type { V1User } from "@rilldata/web-admin/client";
```

Comment:
`web-common` should not import anything from `web-local` or `web-admin`. Now that Rill Developer has more interaction with the admin server, I wonder if we should move the `web-admin/client` to `web-common` and rename `web-admin` to `web-cloud` for clarity. (Though this should certainly happen in a separate PR.)

Reply:
Agreed we should hide Pylon if they're logged out. Two options that come to mind:
1. We simply hide the Contact Rill Support button (and/or use the previous link to community)
2. We keep the Contact Rill Support button but let them email support@rilldata.com instead (we could trigger an email form while requiring them to share an email address)

If trivial, I think option 2 might be better to help build engagement (and we would still retain attribution) but will let Product make the final call.

Reply:
@ericpgreen2 Ya makes sense. Update to not use admin's model. For moving clients to common, I disagree. We can only use those clients on cloud. Any communication has to go through the local server.

@AndrewRTsao Hiding the button for now. We can do a follow up to send an email in cloud as well.

Reply:
It's fine to remove the duplicated entries in the footer